Makefile

```diff
@@ -3,50 +3,46 @@ OS_NAME = $(shell uname -s)
 BUILD_CONFIGURATION = Debug
 BINARIES_PATH = $(shell pwd)/Binaries
 SCRIPTS_PATH = $(shell pwd)/build/scripts
+SRC_PATH = $(shell pwd)/src
+TOOLSET_SRC_PATH = $(shell pwd)/build/MSBuildToolset
 TOOLSET_PATH = $(BINARIES_PATH)/toolset
-RESTORE_SEMAPHORE_PATH = $(TOOLSET_PATH)/restore.semaphore
+RESTORE_SEMAPHORE_PATH = $(BINARIES_PATH)/restore.semaphore
 BOOTSTRAP_PATH = $(BINARIES_PATH)/Bootstrap
 BUILD_LOG_PATH =
 HOME_DIR = $(shell cd ~ && pwd)
 DOTNET_VERSION = 1.0.0-preview2-002911
 NUGET_VERSION = 3.5.0-beta2
 NUGET_EXE = $(shell pwd)/nuget.exe
 
-MSBUILD_ADDITIONALARGS := /v:m /fl /fileloggerparameters:Verbosity=normal /p:Configuration=$(BUILD_CONFIGURATION)
```

Comment:
Woops, I was debugging and forgot to delete that change. `/v:m` is minimal verbosity, not parallel build.